pkg/osbuild/monitor_test.go

```diff
@@ -155,6 +159,27 @@ func TestScannerDuration(t *testing.T) {
 	}, st)
 }
 
+//go:embed testdata/monitor-fedora-42-raw-v159.seq.json
+var osbuildMonitorDuration_fedora []byte
+
+func TestScannerPipelineName(t *testing.T) {
+	scanner := osbuild.NewStatusScanner(bytes.NewBuffer(osbuildMonitorDuration_fedora))
+
+	// the first line is context-less
+	_, err := scanner.Status()
+	assert.NoError(t, err)
+
+	for {
+		st, err := scanner.Status()
+		assert.NoError(t, err)
+		if st == nil {
+			break
+		}
+
+		assert.NotEmpty(t, st.Pipeline)
```

Comment:
I would prefer if the test would be a bit more specific, maybe we can collect all pipeline names and do a `assert.Equal(t, []string{"build", "sources", ...)` here, wdyt?